.github/workflows/pkgdown.yaml

```diff
@@ -1,21 +1,19 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches:
-      - main
-      - master
+    branches: [main, master]
+    tags: ['*']
```

Comment:
Should this be removed?

Reply:
Why? I think we want the site to be re-built when we push a tag (which indicates a release)

Reply:
ok!